CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v2.0.0 - []
+
+### `Added`
+
+- GATK's ShiftFasta to generate all the files required for mitochondrial analysis
+- Feature to calculate CADD scores for indels
+
```

Comment:
Should the title be "added" or does it need to be changed?

Reply:
It is from the template.. as we make changes to an existing feature then we should introduce another heading called `changed` and write such changes under it. For now, I think this is fine 😅

Reply:
Good to know :)